server/routes/notification.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getNotifications,
+  createNotification,
+  getUnReadNotifications,
+  updateNotificationToRead,
+} = require("../controllers/notifications");
+const protect = require("../middleware/auth");
+
+router.route("/").get(protect, getNotifications);
+router.route("/unread/:id").get(protect, getUnReadNotifications);
```

Comment:
```suggestion
router.route("/unread").get(protect, getUnReadNotifications);
```
The `:id` path param isn't being used anywhere, please remove it